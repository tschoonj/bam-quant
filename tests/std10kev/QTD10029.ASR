File: C:\xrfdaten\2014KW30\chris\std10kev\std10029.spe
Start: 21.07.2014 16:06:33
Stop: 21.07.2014 16:08:13
BESSY: 298.4314588 298.4551881
IONI: 1.422787E-10 1.417558E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 66228
Run: 100000
Dead: 34
ICR: 123840
OCR: 0
Accumulated BESSY: 59673.1083437
Accumulated Ioni: 2.84681231E-08
N Accumulation: 200
Normfactor:  1.0607903e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Cu
exp_x 112.020401000977 112.0204
exp_y 122.265808105469 122.2658
exp_z 53.8899993896484 53.89
exp_w 135 135.0001
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  29   1      8.041   7187233.      2682.     158.63
