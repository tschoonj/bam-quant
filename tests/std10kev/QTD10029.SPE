$DATA:
0 2000
15682
10174
4766
1440
338
53
18
19
6
2
4
2
2
1
3
12
126
1455
2113
2008
1841
1838
1735
1629
1597
1536
1490
1447
1455
1408
1381
1337
1319
1178
1207
1196
1167
1202
1180
1135
1098
1133
1118
1065
1045
995
1014
1023
1043
1043
944
941
938
929
998
939
914
1072
919
927
921
888
885
877
930
878
842
919
929
865
875
925
854
870
874
786
799
819
797
814
774
730
824
772
791
786
804
754
787
814
805
786
826
745
762
819
759
756
833
789
815
868
907
862
801
846
797
756
748
796
752
730
748
749
755
716
726
703
676
700
689
720
722
691
710
721
718
710
705
802
814
772
708
687
720
645
712
747
693
716
695
749
804
913
920
1079
1104
1100
996
862
809
759
692
700
678
665
726
683
688
665
654
713
650
709
671
661
666
646
675
639
614
612
675
619
682
621
641
628
644
720
661
643
679
595
633
655
594
646
605
695
635
635
597
633
728
623
666
681
617
631
587
648
624
628
658
616
660
661
654
643
660
656
644
646
663
647
625
651
632
651
704
617
636
646
693
647
566
647
678
676
647
648
708
684
655
657
647
614
642
705
657
718
662
658
667
669
714
690
693
663
656
677
751
696
713
691
700
657
708
714
658
757
743
727
700
761
717
776
773
726
782
729
727
783
769
764
697
712
739
727
746
781
693
783
829
786
810
779
847
817
807
936
848
859
871
882
872
873
889
846
866
912
905
989
1085
1204
1341
1582
1973
2187
2530
2632
2761
2575
2385
2102
1804
1556
1454
1274
1186
1129
1126
1124
1119
1091
1134
1064
1079
1193
1120
1195
1218
1252
1278
1260
1226
1265
1280
1318
1339
1342
1358
1408
1349
1375
1369
1410
1382
1474
1461
1561
1520
1581
1563
1574
1627
1612
1578
1527
1620
1573
1579
1627
1742
1810
1890
1938
1890
2053
2138
2215
2430
2455
2491
2585
2851
2963
3156
3531
3774
4016
4339
4807
5244
6048
7807
10844
17539
31032
56808
102706
175862
280887
415355
565330
705934
807244
844804
810862
711323
570474
421304
284209
176178
100897
53974
27130
13329
7019
3999
2783
2203
2013
2012
1765
1781
1746
1662
1802
1698
1858
1951
2087
2201
2438
2728
2914
3243
3794
4717
6297
8966
13498
21557
32924
47783
66190
85346
102285
111852
114090
107784
93443
74619
55206
37672
23992
14373
7733
4046
1960
1008
514
275
195
134
123
100
107
103
78
92
67
83
75
68
61
74
61
67
67
62
60
63
54
60
60
53
64
87
79
78
92
105
117
121
139
189
218
286
293
283
338
328
349
302
249
290
193
175
147
108
81
55
49
42
25
23
28
27
19
25
17
21
20
19
24
20
23
25
24
26
18
20
23
22
26
22
20
28
22
15
17
13
21
24
28
19
20
23
18
24
12
24
21
27
25
26
20
19
15
18
26
18
19
19
15
18
13
21
17
15
18
12
15
21
18
16
14
21
23
15
16
18
21
20
19
14
12
18
18
13
15
19
18
17
15
15
19
23
14
26
22
16
17
19
13
12
14
14
13
10
10
21
15
11
17
25
11
12
17
17
12
15
13
18
16
21
17
24
16
14
11
14
17
15
12
11
15
20
16
13
13
17
19
12
18
18
13
12
16
14
24
24
18
14
19
13
14
9
18
9
12
13
12
17
16
20
18
17
11
19
18
14
15
6
11
21
21
16
22
14
22
16
17
16
21
14
13
11
16
15
15
15
17
13
12
15
22
15
15
18
25
22
20
8
24
20
18
19
19
35
41
30
47
39
41
32
49
43
53
53
29
36
31
27
35
34
23
30
29
18
23
16
22
22
22
12
32
29
20
24
22
21
24
20
29
24
23
25
34
44
34
35
24
28
23
29
35
46
37
23
27
24
24
33
36
29
29
24
31
27
33
42
52
36
28
39
43
46
40
52
47
63
66
68
82
92
104
102
131
180
311
440
625
918
1394
1806
2534
3122
4047
4743
5525
5888
6200
5981
5845
5171
4678
3828
2970
2209
1630
1157
767
486
306
173
122
91
68
59
38
42
23
50
43
37
42
49
56
62
75
76
111
152
193
255
376
504
622
836
1041
1240
1381
1592
1637
1704
1601
1608
1416
1183
927
790
612
377
318
204
135
80
60
30
20
12
10
9
7
4
4
5
7
4
5
6
10
12
14
12
24
19
23
35
51
49
66
88
84
107
131
99
105
111
109
88
83
58
56
47
29
30
20
15
11
10
4
8
7
3
3
2
2
2
1
2
1
0
2
0
1
1
0
0
0
0
2
0
1
0
0
0
1
0
2
1
1
0
1
0
0
1
2
1
0
2
0
3
1
1
0
0
1
0
2
0
0
0
1
1
0
0
1
0
1
0
0
1
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
1
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
2
0
0
2
0
0
1
0
0
1
0
0
0
0
0
0
0
0
1
0
0
0
0
1
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
2
0
0
0
1
0
0
0
0
2
0
0
0
0
0
1
0
0
0
0
1
0
0
0
0
0
1
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
1
1
0
0
1
0
2
0
0
0
2
0
0
0
0
1
0
0
0
0
0
1
0
0
0
0
0
1
1
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
0
0
1
0
0
0
0
1
0
0
0
2
0
0
1
0
0
0
1
1
0
1
0
2
1
1
5
2
4
10
19
13
13
20
16
29
28
27
39
37
26
25
27
19
19
18
14
11
14
9
9
2
1
1
0
3
1
0
1
2
1
0
1
0
1
1
0
1
1
3
3
1
4
6
3
8
9
10
11
14
11
9
16
16
11
10
7
8
7
11
4
4
2
0
5
0
0
1
0
0
0
0
0
0
0
0
0
0
1
0
1
1
0
0
1
1
0
0
2
1
0
1
1
0
1
1
4
2
2
2
1
3
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
2
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std10kev\std10029.spe
Start: 21.07.2014 16:06:33
Stop: 21.07.2014 16:08:13
BESSY: 298.4314588 298.4551881
IONI: 1.422787E-10 1.417558E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 66228
Run: 100000
Dead: 34
ICR: 123840
OCR: 0
Accumulated BESSY: 59673.1083437
Accumulated Ioni: 2.84681231E-08
N Accumulation: 200
Normfactor:  1.0607903e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Cu
 
exp_x 112.020401000977 112.0204
exp_y 122.265808105469 122.2658
exp_z 53.8899993896484 53.89
exp_w 135 135.0001
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  29   1      8.041   7187233.      2682.     158.63
