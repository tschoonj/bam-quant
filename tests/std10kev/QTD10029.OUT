     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:27:04  บ   
     บ  Spectrum: QTD10029.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 - 500        5 iterations done    บ   
     บ  ChiSquare =   419.9   last change =   .00%    lambda= 1.E-07   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -48.0 ๑   100.0   บ     -48.0 ๑      .7   บ   
     บ  GAIN (eV/ch)  บ     20.274 ๑   2.000   บ    20.274 ๑    .002   บ   
     บ det NOISE (eV) บ     -124.8 ๑    40.0   บ    -124.8 ๑    -2.6   บ   
     บ  FANO factor   บ       .245 ๑    .050   บ      .245 ๑    .003   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Cu-Ka                           7187233. ๑   2682.           บ   
     บ    KA1         8.048    .66084     4749633. ๑   1773.    177.2  บ   
     บ              399.328    163.11        3638.           9.57E-01  บ   
     บ    KA2         8.028    .33916     2437600. ๑    912.    158.6  บ   
     บ              398.342    162.79        3684.           9.57E-01  บ   
     บ                                                                 บ   
     บ  2 Cu-Kb                           1033225. ๑   1018.           บ   
     บ    KB1         8.905   1.00000     1033225. ๑   1019.    101.1  บ   
     บ              441.599    176.34        2155.           9.68E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
